Peer/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Peer.ConnectorApi;
+using Peer.Connectors;
+using Peer.Domain;
+
+namespace Peer
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var services = new ServiceCollection();
+
+            services.AddTransient<IPullRequestSource, GithubPullRequestSource>();
+
+            var sp = services.BuildServiceProvider();
+
```

Comment:
Does this work atm? You aren't registering the implementation of IClient that the github api needs